go.mod

```diff
@@ -352,7 +352,7 @@ replace (
 	github.com/cosmos/cosmos-sdk => ./sei-cosmos
 	github.com/cosmos/iavl => github.com/sei-protocol/sei-iavl v0.2.0
 	github.com/cosmos/ibc-go/v3 => github.com/sei-protocol/sei-ibc-go/v3 v3.3.6
-	github.com/ethereum/go-ethereum => github.com/sei-protocol/go-ethereum v1.15.7-sei-7.0.20250929182230-93350978bb7c
+	github.com/ethereum/go-ethereum => github.com/sei-protocol/go-ethereum v1.15.7-sei-9
```

Comment:
Just to pick your brain @codchen, would it be useful to move go-ethereum into this repo too?

Reply:
Thank you both. I think it's best to keep go-ethereum repo separate at least for now. I will integrate the UCI versioning into it though. That should make the version bumps reviewable/easier and provide a better structure for bubble up.

I think we can automate the bubble up via Dependabot entirely for that repo at least. First things first: semver so that the bot can infer what's newer.